Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/http-utils.ts

```diff
@@ -33,15 +38,17 @@ const getErrorMessage = (
 
         return defaultErrorMessage;
     }
+
     if ('status' in err) {
         return 'error' in err
             ? err.error.replace(/"/g, '')
             : ((err as any).data as IErrorDataType).detail.replace(/"/g, '');
     }
 
-    if (err.message) {
-        return err.message.replace(/"/g, '');
+    if ((err as SerializedError).message) {
+        return (err as SerializedError).message!.replace(/"/g, '');
```

Comment:
Could we use type narrowing in order to improve ```code readability```? For example, we could use:
```
if (err instanceof SerializedError && err.message) {
    return err.message.replace(/"/g, '');
}
```

Reply:
Typescript does not allow this:
TS2693: SerializedError only refers to a type, but is being used as a value here.